feature/platform/transceiver/zr_logical_channels_test/README.md

```diff
@@ -0,0 +1,98 @@
+# TRANSCEIVER-11: Telemetry: 400ZR Optics logical channels provisioning and related telemetry.
+
+## Summary
+
+Routing devices that support transceivers with built-in DSPs like 400ZR consume
+the [OC-terminal-device model](https://openconfig.net/projects/models/schemadocs/jstree/openconfig-terminal-device.html)
+model.
+The ZR signal in these transceivers traverses through a series of
+terminal-device/logical-channels. The series of logical-channel utilizes the
+assignment/optical-channel leaf to create the relationship to
+OPTICAL_CHANNEL. For 400ZR 1x400GE mode this heirarchy looks like:
+400GE Eth. Logical Channel => 400G Coherent Logical Channel => OPTICAL_CHANNEL
+Purpose of this test is to verify the logical channel provisioning and related
+telemetry.
+
+## Procedure
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module.
```

Comment:
ZR needs to connects to a ZR module , it can not connect to 400GE port on the ATE like Ixia unless Ixia can host a 400ZR module. 

Updated the text in the test as follows:

Connect two ZR interfaces using a duplex LC fiber jumper such that TX
output power of one is the RX input power of the other module. The A and Z
ends of the link should be the same 400ZR PMD. Optics can be connected
through passive patch panels or an optical switch as needed, as long as the
overall link loss budget is kept under 2 - 3 dB. There is no requirement to 
deploy a separate line system for functional tests.